can/interfaces/canal/canalInterface.py

```diff
@@ -0,0 +1,196 @@
+# this interface is for windows only, otherwise use socketCAN
+
+import logging
+
+from can import BusABC, Message
+from can.interfaces.canal.canal_wrapper import *
+
+bootTimeEpoch = 0
+try:
+    import uptime
```

Comment:
This kind of optional dependency should be documented in the interface docs

Reply:
Actually this can be completely removed since I was not able to find any usage of  `bootTimeEpoch`.